arcgis-ios-sdk-samples/Utility network/Trace utility network/TraceUtilityNetworkViewController.swift

```diff
@@ -83,17 +54,54 @@ class TraceUtilityNetworkViewController: UIViewController, AGSGeoViewTouchDelega
         return overlay
     }()
     
-    // MARK: Initialize map and Utility Network
+    // MARK: Initialize map, utility network, and service geodatabase
     required init?(coder aDecoder: NSCoder) {
         // Create the map
         map = AGSMap(basemapStyle: .arcGISStreetsNight)
+        
         // Create the utility network, referencing the map.
         utilityNetwork = AGSUtilityNetwork(url: featureServiceURL, map: map)
-        
+        // Create the service geodatabase that matches the utility network.
+        serviceGeodatabase = AGSServiceGeodatabase(url: featureServiceURL)
         super.init(coder: aDecoder)
+        // Load the service geodatabase.
+        serviceGeodatabase.load { [weak self] error in
```

Comment:
```suggestion
        serviceGeodatabase.load { [weak self] _ in
```
Since the error is not checked here, replace it with underscore.